### PR TITLE
bbn-test-5: Add post-upgrade snapshot

### PR DESCRIPTION
**Pull request by filippos47**
Also, make some fixes / updates on the node setup doc.

### CODE REVIEW

File: bbn-test-5/babylon-node/README.md [hunk 1]
```diff
@@ -98,10 +98,10 @@ Parameters:
 - `minimum-gas-prices`: The minimum gas price (in this example we use `0.005ubbn`)
    that your node will accept for transactions. Transactions with lower gas 
    prices will be rejected.
-- `iavl-cache-size`: Default is "5000" (uses ~3GB of memory). Setting to 
+- `iavl-cache-size`: Default is "781250". Setting to 
    0 disables the IAVL tree caching, which reduces memory usage but significantly
```
Comment (vitsalis) [inline bbn-test-5/babylon-node/README.md:102]: Why do we recommend people to have a default with lower performance?

File: bbn-test-5/babylon-node/README.md [hunk 2]
```diff
@@ -111,30 +111,39 @@ Note: If you're running a validator or RPC node that needs to handle queries,
...
 ```
 
 Parameters:
 - `seeds`: Comma separated list of seed nodes that your node will connect to for 
-discovering other peers in the network
+discovering other peers in the network; you can obtain seed endpoints from
+[here](../README.md#seed-nodes)
+- `persistent_peers`: Comma separated list of nodes that your node will use as
+persistent peers; you can obtain peers from [here](../README.md#peers)
+- `timeout_commit`: The Babylon network block time
```
Comment (vitsalis) [inline bbn-test-5/babylon-node/README.md:136]: Let's highlight this somehow? That the user must set it

**Review by vitsalis — APPROVED** [review]

File: bbn-test-5/babylon-node/README.md (hunk 1 above)
Comment (filippos47) [reply bbn-test-5/babylon-node/README.md:102]: Enabling the IAVL cache will exhaust the available memory, even on 128G machines. We need to dig deeper to understand why.

Until then, we decided to turn off the IAVL cache - it won't have a performance impact to regular users.

File: bbn-test-5/babylon-node/README.md (hunk 2 above)
Comment (filippos47) [reply bbn-test-5/babylon-node/README.md]: sure

**Review by vitsalis — APPROVED** [review]

**Pull request merged**